plots/electricity_bills.py

```diff
@@ -0,0 +1,255 @@
+# relative path to folder where to store plots
+PATH_PLOTS = "plots/results/"
+
+def change_path_to_pypsa_eur():
+    # path to pypsa-eur
+    pypsa_path = "submodules/pypsa-eur/"
+    # absolute path to pypsa-eur
+    new_path = os.path.join(BASE_PATH, pypsa_path)
+    # change path to pypsa-eur
+    os.chdir(new_path)
+
+
+def load_networks(lineex, space_resolution, sector_opts, planning):
+    FILE = f"elec_s_{space_resolution}_l{lineex}__{sector_opts}_{planning}_flexible.nc"
+    DIR = "results/flexible/postnetworks"
+    n_flex = pypsa.Network(os.path.join(DIR, FILE))
+
+    FILE = f"elec_s_{space_resolution}_l{lineex}__{sector_opts}_{planning}_retro_tes.nc"
+    DIR = "results/retro_tes/postnetworks"
+    n_retro_tes = pypsa.Network(os.path.join(DIR, FILE))
+
+    FILE = f"elec_s_{space_resolution}_l{lineex}__{sector_opts}_{planning}_flexible-moderate.nc"
+    DIR = "results/flexible-moderate/postnetworks"
+    n_flex_mod = pypsa.Network(os.path.join(DIR, FILE))
+
+    FILE = f"elec_s_{space_resolution}_l{lineex}__{sector_opts}_{planning}_rigid.nc"
+    DIR = "results/rigid/postnetworks"
+    n_rigid = pypsa.Network(os.path.join(DIR, FILE))
+
+    return n_flex, n_flex_mod, n_retro_tes, n_rigid
+
+
+def change_path_to_base():
+    os.chdir(BASE_PATH)
+    # create folder to store images
+    os.makedirs(PATH_PLOTS, exist_ok=True)
+
+
+def get_households():
+    household_data = pd.read_excel("plots/Households.xlsx",skiprows=1,usecols="B:D")
+    households = household_data.set_index("Country")[ "Households (thousands)"]
+    households.index.name=""
+    return households
+
+
+def electricity_bills(network, households):
+    n = network.copy()
+    
+    rh_techs_elec = ['residential rural ground heat pump',
+                     'residential rural resistive heater', 
+                     'residential urban decentral air heat pump',
+                     'residential urban decentral resistive heater']
+    rh_techs_gas = ['residential rural gas boiler', 'residential urban decentral gas boiler']
+    rh_techs_mCHP = ['residential rural micro gas CHP', 'residential urban decentral micro gas CHP']
+    
+    ev_tech_charge = ['BEV charger']
+    ev_tech_discharge = ["V2G"]
+    
+    # map loads to corresponding buses (AL1 0 in load to AL1 0 low voltage)
+    n.loads_t.p_set =  n.loads_t.p_set.rename(columns=n.loads.bus.to_dict())
+    
+    # low voltage load
+    lv_buses = n.buses.query("carrier == 'low voltage'").index
+    lv_load = n.loads_t.p_set[lv_buses]
+    
+    # electricity consumption of residential heat techs in low_voltage bus of links
+    rh_elec_links = n.links.query("carrier in @rh_techs_elec").index
+    rh_techs_mapping = n.links.loc[rh_elec_links, "bus0"].to_dict()
+    rh_techs_consume = n.links_t.p0[rh_elec_links]
+    rh_techs_consume = rh_techs_consume.rename(columns=rh_techs_mapping)
+    rh_techs_consume = rh_techs_consume.groupby(level=0, axis=1).sum()
```

Comment:
```suggestion
    rh_gas_consume.T.groupby(level=0).sum().T
```
`axis` is deprecated, pls use `.T`